lib/chef/resource/sysctl.rb

```diff
@@ -44,7 +104,7 @@ class Sysctl < Chef::Resource
       property :value, [Array, String, Integer, Float],
         description: "The value to set.",
         coerce: proc { |v| coerce_value(v) },
-        required: true
+        required: %i(:apply)
```

Comment:
```suggestion
        required: [:apply]
```

Was this intentional?

```
irb(main):001:0> %i{foo}
=> [:foo]
irb(main):002:0> %i{:foo}
=> [:":foo"]
```

Reply:
Just a typo